openslides/core/management/commands/getgeiss.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+import stat
+import sys
+from urllib.request import urlopen, urlretrieve
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """
+    Command to get the latest release of geiss from github.
+    """
+    help = 'Get the latest geiss release from github.'
+
+    def handle(self, *args, **options):
+        response = urlopen('https://api.github.com/repos/ostcar/geiss/releases/latest').read()
+        release = json.loads(response.decode())
+        geiss_name = get_geiss_name()
+        download_url = ""
```

Comment:
Instead of setting an empty string you can do so if you like:
```python
for asset in release['assets']:
    if asset['name'] == geiss_name:
        download_url = asset['browser_download_url']
        break
else:
    raise CommandError("Could not find download url in release.")
```